pages/checklist-plots.py

```diff
@@ -0,0 +1,13 @@
+from utils.code_and_show import example_app
+
+dash.register_page(
+￼    __name__,
+￼    description = "Dash Sample App",
+￼    layout_type = "1 Row 1 Col",
+￼    graph = "scatter",
+    components = "checklist",
+￼    callback = "1input:1output",
+    dataset = "iris"
+￼)
+￼
+￼filename = __name__.split("pages.")[1]
```

Comment:
I'm not 100% sure, but I think you need to add this line at the very end, as in [this example](https://github.com/AnnMarieW/dash-app-gallery/blob/main/pages/axes.py):
```
def layout():
    return example_app(f"pages/examples/{filename}.py")
```